ballerina/encrypt_decrypt.bal

```diff
@@ -243,3 +243,41 @@ public isolated function decryptAesGcm(byte[] input, byte[] key, byte[] iv, AesP
     name: "decryptAesGcm",
     'class: "io.ballerina.stdlib.crypto.nativeimpl.Decrypt"
 } external;
+
+# Returns the PGP-encrypted value for the given data.
+# ```ballerina
+# byte[] message = "Hello Ballerina!".toBytes();
+# byte[] publicKey = check io:fileReadBytes("public_key.asc"); // provide the path to the public key
+# byte[] cipherText = check crypto:encryptPgp(message, publicKey);
+# ```
+#
+# + plainText - The content to be encrypted
+# + publicKey - Public key
+# + options - PGP encryption options
+# + return - Encrypted data or else a `crypto:Error` if the key is invalid
+public isolated function encryptPgp(byte[] plainText, byte[] publicKey, *PgpEncryptionOptions options)
```

Comment:
Let's take in the `publicKey` and `privateKey` as file paths for the moment.

Reply:
The same should applied for the params `privateKey` and `passphrase` in `decryptPgp` as well right? Should we keep the return type to `byte []` as it is for those two methods `encryptPgp` and `decryptPgp`?

Reply:
Let's have `privateKey` as a string file path in `decryptPgp` function as well. The passphrase should be a byte[] and what we return from the APIs also should be byte[] s.

Reply:
I have addressed this in https://github.com/ballerina-platform/module-ballerina-crypto/pull/555/commits/78e3fb4fbeab147e5c59d3a51cfec372b405651c